Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:8
```

Comment:
Try using `node:8-alpine`, it's a lot smaller :) Should work for our purposes

Reply:
ah cool

Reply:
did not know